lib/widgets/message_list.dart

```diff
@@ -86,7 +84,7 @@ class MessageListTheme extends ThemeExtension<MessageListTheme> {
     required this.dateSeparator,
     required this.dateSeparatorText,
     required this.dmRecipientHeaderBg,
-    required this.editedMovedMarkerCollapsed,
+    required this.editStateLabel,
```

Comment:
It looks like there's a Figma design variable for this, `labelEdited`, so let's use that and put it in `DesignVariables`, sorted alphabetically in the default section (not marked `// Not exactly from the Figma design […]` or `// Not named variables in Figma […]`).